app/javascript/packages/verify-flow/steps/password-confirm/forgot-password.tsx

```diff
@@ -0,0 +1,45 @@
+import { useContext } from 'react';
+import { PageHeading, Button } from '@18f/identity-components';
+import { t } from '@18f/identity-i18n';
+import { getAssetPath } from '@18f/identity-assets';
+import { VerifyFlowContext } from '@18f/identity-verify-flow';
+
+interface ForgotPasswordProps {
+  goBack: () => void;
+}
+
+export function ForgotPassword({ goBack }: ForgotPasswordProps) {
+  const { resetPasswordUrl } = useContext(VerifyFlowContext);
+
+  function goToResetPassword() {
+    window.location.href = resetPasswordUrl!;
+  }
+
+  return (
+    <>
+      <img
+        src={getAssetPath('status/info-question.svg')}
+        width="50"
+        alt={t('components.status_page.icons.question')}
+        height="50"
+        className="margin-bottom-4"
+      />
+      <PageHeading>{t('idv.forgot_password.modal_header')}</PageHeading>
+      <ul className="usa-list">
+        {t(['idv.forgot_password.warnings']).map((warning) => (
```

Comment:
I just realized the new array support won't work without corresponding updates to [the Webpack plugin](https://github.com/18F/identity-idp/tree/main/app/javascript/packages/rails-i18n-webpack-plugin) that detects strings being used. Currently, I see plaintext message:

![image](https://user-images.githubusercontent.com/1779930/168671411-91584509-feba-4231-a209-2291912f29d7.png)

Kinda non-ideal, but a short-term solution could be to force TypeScript to pick up the singular key as returning an array.

```suggestion
        {(t('idv.forgot_password.warnings') as unknown as string[]).map((warning) => (
```

Reply:
> I just realized the new array support won't work without corresponding updates to [the Webpack plugin](https://github.com/18F/identity-idp/tree/main/app/javascript/packages/rails-i18n-webpack-plugin) that detects strings being used.

Follow-up at #6359